### PR TITLE
Navigation markup update

### PR DESCRIPTION
**Pull request by wengole**

### CODE REVIEW

**Comment by pbauer** [comment]
This is already 29 days old. Is it obsolete or ready to merge? @wengole can you comment and is ask someone to review please?

**Comment by adrianwhite** [comment]
@pbauer @wengole the markup was correct at the time that this was done.
Cornelis has since added a bunch of functionlity that has not been implemented in the backend and a few other modifications have been made.

What should we do with this now?
 1. Merge it as it is (relevant to completeing the issue @wengole was working on)?
 2. Update with the latest markup without the latest additions to functionality?
 3. Update with the latest markup with the latest additions that the back-end does not support but comment it out?

@pilz or @gyst do you have a suggestion on this?

File: src/ploneintranet/workspace/browser/tiles/navigation.py [hunk 1]
```diff
@@ -56,4 +53,48 @@ def render(self):
...
+        """
+        Is the current node the root of the workspace?
+        """
+        return IWorkspaceFolder.providedBy(self.root)
+
+    def parent_title(self):
+        return self._parent().Title()
+
+    def non_folderish_item_uids(self):
+        # TODO: Better way of determining folderish?
```
Comment (adamcheasley) [inline src/ploneintranet/workspace/browser/tiles/navigation.py]: It appears you should be able to use the `isPrincipiaFolderish` attribute/index:
http://stackoverflow.com/questions/15177286/how-to-add-the-is-folderish-attribute-to-dexterity-objects

**Comment by mattss** [comment]
Navigation markup has changed significantly. Closing PR for now.

**Pull request closed without merging**